.gitignore

```diff
@@ -16,6 +16,9 @@
 # Ignore node_modules
 node_modules
 
+# ignore makefile with secret npm token
+Makefile
```

Comment:
Other developers may still need to use this file. Is there an alternative to removing it from Git?